forgettable-frontend/src/components/InputField/InputField.module.css

```diff
@@ -0,0 +1,55 @@
+.inputLabelPrimary {
+    font-family: 'Poppins', sans-serif;
+    font-size: 23px;
+    font-weight: bold;
+    text-align: left;
+    display: flex;
+    flex-direction: column;
+}
+
+.inputLabelSecondary {
+    font-family: 'Poppins', sans-serif;
```

Comment:
the font-family is set in the top level css, so you shouldn't have to set this. leaving it unset here will allow us to easily globally change the font if we change our mind on the font

Reply:
I can remove the font-family declaration and have the same font except in the multi-line input field. In the interest of font consistency, I will keep the declaration for this specific type of input field.

_Multi-line font with font-family declaration:_
![image](https://user-images.githubusercontent.com/68996358/158726800-a3ac5ada-b930-4bf9-af80-88c4f9e4b866.png)

_Multi-line font without font-family declaration:_
![image](https://user-images.githubusercontent.com/68996358/158726943-b003823d-a5ae-478c-9d03-25951fd5cb07.png)